scripts/helpers/restyle-diff.sh

```diff
@@ -50,3 +50,9 @@ fi
 
 declare -a paths="($(git diff --ignore-submodules --name-only --merge-base "$ref"))"
 restyle-paths "${paths[@]}"
+
+if [ ! -f "/tmp/ktfmt-0.44-jar-with-dependencies.jar" ]; then
+    wget "https://repo1.maven.org/maven2/com/facebook/ktfmt/0.44/ktfmt-0.44-jar-with-dependencies.jar" -P /tmp
+fi
+
+find src examples -name '*.kt' | xargs java -jar /tmp/ktfmt-0.44-jar-with-dependencies.jar --google-style
```

Comment:
This is not restyling the right set of files.  restyle-diff is supposed to restyle the files that differe from the base.

Also, this will break if "java" is not installed, even if nothing java-related is in the diff.

Reply:
Acked,  I separate kotlin format to a new shell script kotlin-format.sh, this script conducts the same reformat to conform the existing kotlin format check in CI